helm-charts/FATE/templates/core/python-spark.yaml

```diff
@@ -34,6 +34,7 @@ spec:
         fateMoudle: python
 {{ include "fate.labels" . | indent 8 }}
     spec:
+      serviceAccountName: flow
       {{- if .Values.istio.enabled }}
```

Comment:
We exposed this settings in cluster.yaml, see Line ~198. Maybe change line 198~200 to:

```
    {{- if .Values.modules.python.serviceAccountName }}
      serviceAccountName: {{ .Values.modules.python.serviceAccountName }}
    {{- else }}
      serviceAccountName: flow
    {{- end }}
```

And change the rbac.yaml's service account name accordingly too.

Reply:
Fixed, TFTR